python/sglang/srt/layers/quantization/modelopt_quant.py

```diff
@@ -1003,3 +1005,5 @@ def apply(
             params=layer.cutlass_moe_params,
             apply_router_weight_on_input=apply_router_weight_on_input,
         ).to(x.dtype)
+        output *= routed_scaling_factor
```

Comment:
Can we fuse this multiplication into any operators?

Reply:
We might be able to fuse it into select experts. Let me try.

Reply:
I was able to fuse it. I opened a follow up PR: https://github.com/sgl-project/sglang/pull/8364